src/Components/DialPad/dialPad.styles.scss

```diff
@@ -0,0 +1,78 @@
+.container {
```

Comment:
Please make sure that the design is responsive

Reply:
ok , I will do .